packages/components/Notification/src/Notification.styling.ts

```diff
@@ -102,19 +100,5 @@ export const stylingSettings: UseStylingOptions<NotificationProps, NotificationS
       },
       ['color', ...fontStyles.keys],
     ),
-    dismissIcon: buildProps(
-      (tokens: NotificationTokens) => {
-        return {
-          style: {
-            alignSelf: 'center',
-            flex: 1,
-            height: 20,
-            width: 20,
-          },
-          color: tokens.color,
-        };
-      },
-      ['color'],
-    ),
```

Comment:
why did the go away?

Reply:
Button has the option to be an icon only button so I thought that would be more useful than having a svg inside an empty button. Also the old way meant that the svg wouldn't change colors when pressed.